students/Hebdowska_Grazyna/lesson_04_uni_testing/unit_testing_fibonacci.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+
+def fib(number):
+    if number == 0:
+        return 0
+    elif number == 1:
+        return 1
+    else:
+        return fib(number - 1) + fib(number - 2)
+
+
+class MyTest(unittest.TestCase):
+
+    def test_fibonaci_none(self):
```

Comment:
Please try to use pytest for testing Fibonacci as it is the most common lib for writing unittest in python world
Example
```
    def test_first(self):
        assert fib(1) == 1
```